starcli/layouts.py

```diff
@@ -36,7 +36,7 @@ def shorten_count(number):
 def get_stats(repo):
     """ return formatted string of repo stats """
     stats = f"{repo['stargazers_count']} ⭐ " if repo["stargazers_count"] != "-1" else ""
```

Comment:
After you've done that we can just make these if repo[xx] else None
Instead of comparing it to -1